src/main/java/lutchat/Storage.java

```diff
@@ -0,0 +1,87 @@
+package lutchat;
+
+import java.io.File;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> taskList = new ArrayList<>();
+
+
+        File file = new File(filePath);
+        file.getParentFile().mkdirs();
+        if (!file.exists()) {
+            file.createNewFile();
+            return taskList;
+        }
+
+        Scanner sc = new Scanner(file);
+        while (sc.hasNextLine()) {
+            String data = sc.nextLine();
+            String[] parts = data.split(" \\| ");
+            if (parts.length < 3) {
+                System.out.println("Skipping corrupted data: " + data);
+                continue;
+            }
+
+            String taskType = parts[0];
+            boolean done = parts[1].equals("1");
+            String desc = parts[2];
+
+            Task task = null;
+            switch (taskType) {
```

Comment:
Switch case does not follow the coding standard. There should not be an indent for the switch case.